docs/topics/gradle/gradle-configure-project.md

```diff
@@ -164,7 +150,6 @@ for tasks that depend on a major JDK version.
 The Kotlin Gradle plugin supports Java toolchains for Kotlin/JVM compilation tasks. JS and Native tasks don't use toolchains.
 The Kotlin compiler always runs on the JDK the Gradle daemon is running on.
 A Java toolchain:
-* Sets the [`jdkHome` option](gradle-compiler-options.md#attributes-specific-to-jvm) available for JVM targets.
```

Comment:
Hm, it is still true but link should point to compiler arguments (CLI) and not Gradle one